src/System Application/App/Cryptography Management/src/SignatureAlgorithm.Enum.al

```diff
@@ -42,6 +42,19 @@ enum 1446 SignatureAlgorithm implements "Signature Algorithm v2"
                             "Signature Algorithm v2" = "DSACryptoServiceProvider Impl.";
 #else
         Implementation = "Signature Algorithm v2" = "DSACryptoServiceProvider Impl.";
+#endif
+    }
+    /// <summary>
+    /// Specifies the RSASSA-PSS algorithm implemented by RSA
+    /// </summary>
+    value(2; "RSASSA-PSS")
+    {
+
+#if not CLEAN24
```

Comment:
Also remove the unnecessary code. Should directly be Signature.. v2 = RSA Impl

Reply:
Hi  @darjoo 
I would ideally need the change for BC23. That is the reason why I have implemented the old interface. Is this basically possible or would I have to proceed completely differently?

Reply:
We can't get this in for v23. The earliest would be a minor in v24.
As this is a new feature and not an issue that needs to be hotfixed.

Reply:
@MattiLE, by default, new PRs make it into the upcoming major release. Upon request, I will always happily backport it to the upcoming minor. Only if easily possible, will I backport to the upcoming minor of a past release as well (but that will be the exception). But let's start with the major version and take it from there! 🥳